devito/ir/support/space.py

```diff
@@ -259,6 +259,11 @@ def negate(self):
     def zero(self):
         return Interval(self.dim, 0, 0, self.stamp)
 
+    def ceil(self, o):
```

Comment:
I would turn the API into `ceil(self, v)` where `v` is just an integer. It doesn't make much sense to me to "ceil" based on an Interval